src/main/java/com/mathworks/ci/RunMatlabTestsBuilder.java

```diff
@@ -301,7 +301,8 @@ private int execMatlabCommand(FilePath workspace, Launcher launcher,
     public String constructCommandForTest(FilePath scriptPath) {
         final String matlabScriptName = getValidMatlabFileName(scriptPath.getBaseName());
         final String runCommand = "addpath('" + scriptPath.getRemote().replaceAll("'", "''")
-                + "'); " + matlabScriptName;
+                + "'); " + matlabScriptName + ",delete .matlab/" + scriptPath.getBaseName() + "/"
+                + matlabScriptName + ".m,run(runner_script),rmdir(tmpdir,'s')";

```

Comment:
Same comment as above, use the function form of `delete`.

Reply:
does delete remove Directory ?